src/start_fact_db.py

```diff
@@ -31,12 +30,8 @@ class FactDb(FactBase):
 
     def __init__(self):
         _, config = program_setup(self.PROGRAM_NAME, self.PROGRAM_DESCRIPTION, self.COMPONENT)
-        self.mongo_server = MongoMgr(config=config)
         super().__init__()
-
-    def shutdown(self):
-        super().shutdown()
-        self.mongo_server.shutdown()
+        # FixMe postgres runs as a service. Is this script still useful?
```

Comment:
Hm. Looks like we're not getting status info from the machine when this is not running hm. We probably want that. Is it sensible to start postgres and redis directly instead of via upstart / sysV / ..? I mean we did that with mongo. It allows to run FACT more independent from other stuff on the system.

Reply:
It seems postgres normally only runs as a service. It seems to be possible to start Redis directly but `redis-server some-redis.conf --daemonize no` directly crashes for me.

Reply:
It would be possible to run both in docker containers, though